compiler/rustc_const_eval/src/const_eval/valtrees.rs

```diff
@@ -411,9 +411,9 @@ fn valtree_into_mplace<'tcx>(
                         debug!(?unsized_inner_ty);
 
                         let inner_ty = match ty.kind() {
-                            ty::Adt(def, substs) => {
-                                def.variant(FIRST_VARIANT).fields[i].ty(tcx, substs)
-                            }
+                            ty::Adt(def, substs) => def.variant(FIRST_VARIANT).fields
+                                [FieldIdx::from_usize(i)]
+                            .ty(tcx, substs),
```

Comment:
this seems like a wonky formatting?

Reply:
Hmm, maybe I'll do some shadowing for these cases to try to get something less weird...

Reply:
Sometimes rustfmt just does literally no formatting, for example when there are if-let-chains, maybe this is the case here.

Reply:
Nope, this is *definitely* `rustfmt` doing it -- all I did was add the `FieldIdx::from_usize` inline, without adding any newlines.  Then tidy made me change it to that weirdness 🤷